napari/utils/context/_tests/test_expressions.py

```diff
@@ -0,0 +1,198 @@
+import ast
+
+import pytest
+
+from napari.utils.context._expressions import (
+    _OPS,
+    Constant,
+    Expr,
+    Name,
+    _iter_names,
+    parse_expression,
+)
+
+
+def test_names():
+    assert Name("n").eval({'n': 5}) == 5
+
+    # currently, evaludating with a missing name is an error.
+    with pytest.raises(NameError):
+        Name("n").eval()
+
+    assert repr(Name('n')) == "Name(id='n', ctx=Load())"
+
+
+def test_constants():
+    assert Constant(1).eval() == 1
+    assert Constant(3.14).eval() == 3.14
+    assert Constant('asdf').eval() == 'asdf'
+    assert Constant(b'byte').eval() == b'byte'
+    assert str(Constant(b'byte')) == "b'byte'"
```

Comment:
do we want to be stricter than python and prevent some of the string prefixes like `u`, or `r` ?

Reply:
sure, seems like a reasonable starting point